packages/substrate-client/src/chainhead/storage.ts

```diff
@@ -26,10 +26,10 @@ export const createStorageFn = (
     let result: any = isDescendants ? [] : null
 
     const onItems = isDescendants
-      ? (items: StorageItemResponse[]) => {
+      ? (items: StorageItemResponseRpc[]) => {
```

Comment:
this doesnt matter b/c they are the same type but should this be `StorageItemResponse`? Since the signature on `FollowResponse` is `onItems: (items: Array<StorageItemResponse>)`

Reply:
this is super nitpicking ofc but we can do this so theres no confusion

```
const onItems: Parameters<typeof cbStore>[3]
```